lonestar/analytics/CMakeLists.txt

```diff
@@ -1,9 +1,29 @@
+function(app_analy_cpu name bin_name)
+  app_cpu("ANALYTICS_CPU_" ${name} ${bin_name})
+endfunction()
+
+function(app_analy_dist name bin_name)
+  set(options NO_GPU)
+  set(one_value_args)
+  set(multi_value_args)
+  cmake_parse_arguments(X "${options}" "${one_value_args}" "${multi_value_args}" ${ARGN})
+  if(${X_NO_GPU})
+    app_dist("ANALYTICS_DIST_" ${name} ${bin_name} NO_GPU)
+  else()
+    app_dist("ANALYTICS_DIST_" ${name} ${bin_name})
+  endif()
+endfunction()
+
+function(app_analy_gpu name bin_name)
+  app_gpu("ANALYTICS_GPU_" ${name} ${bin_name})
+endfunction()
+
 add_subdirectory(cpu)
 
 if(GALOIS_ENABLE_DIST)
   add_subdirectory(distributed)
 endif()
 
-if(GALOIS_ENABLE_GPU)
+if(GALOIS_CUDA_CAPABILITY)
```

Comment:
Use GALOIS_ENABLE_GPU here. 

GALOIS_ENABLE_GPU CMake variable still exists. It is not set by the user, but derived using GALOIS_CUDA_CAPABILITY.